src/content/docs/errors-inbox/legacyerrors-eol.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: 'Errors (legacy) EOL'
+metaDescription: "The errors inbox is transitioning to an improved errors experience starting on January 31st 2024."
+freshnessValidatedDate: never
+---
+import errorsinboxErrorsEol from 'images/errors-inbox_screenshot-full_errors-eol.webp'
+
+We're excited to introduce a new and improved errors experience, designed to provide a more comprehensive and intuitive way to manage errors in your applications. To ensure a smooth transition, please take note of the following important updates:
+
+## Important dates
+
+* **January 31, 2024:** The legacy errors experience in APM will be discontinued and replaced by an improved experience. 
```

Comment:
Can change this to "We will discontinue the legacy errors experience in APM and replace it with an improved experience." to get rid of the passive voice.